### ISSUE TITLE
Rebuilding seems to happen several times at startup

### ISSUE THREAD

**Issue by toddw** [TrueCar/gluestick#626]
# Bug
When starting a gluestick app it re-bundles the app several times. It looks like it might be happening because the re-bundled triggers a new bundle. Eventually, however, it does stop but it seems like it might be a bug

## Reproduction steps
1. `gluestick new gs-rebundle --dev ../gluestick/packages/gluestick`
2. `gluestick start`

## Current Result
```
…
…
[GlueStick][start-server] File /Users/twilliams/dev/gs-v1/build/assets/webpack-chunks.json changed [change]

Starting to optimize CSS...
webpack built 030804976462ac97d2ed in 137ms
[GlueStick][start-server] File /Users/twilliams/dev/gs-v1/build/assets/webpack-chunks.json changed [change]
webpack building...

Starting to optimize CSS...

Starting to optimize CSS...
webpack built 030804976462ac97d2ed in 126ms

Starting to optimize CSS...
[GlueStick][start-server] Building server entry.
[GlueStick][start-server] Building server entry.
[GlueStick][start-server] File /Users/twilliams/dev/gs-v1/build/assets/webpack-chunks.json changed [change]
webpack building...

Starting to optimize CSS...
webpack built 030804976462ac97d2ed in 125ms
[GlueStick][start-server] File /Users/twilliams/dev/gs-v1/build/assets/webpack-chunks.json changed [change]
webpack building...

Starting to optimize CSS...
[GlueStick][start-server] Renderer listening on port 8880.
webpack built 030804976462ac97d2ed in 154ms
[GlueStick][start-server] File /Users/twilliams/dev/gs-v1/build/assets/webpack-chunks.json changed [change]
webpack building...

Starting to optimize CSS...
webpack built 030804976462ac97d2ed in 117ms

Starting to optimize CSS...
[GlueStick][start-server] File /Users/twilliams/dev/gs-v1/build/assets/webpack-chunks.json changed [change]
[GlueStick][start-server] Building server entry.
webpack building...

Starting to optimize CSS...

Starting to optimize CSS...
webpack built 030804976462ac97d2ed in 122ms
[GlueStick][start-server] File /Users/twilliams/dev/gs-v1/build/assets/webpack-chunks.json changed [change]
[GlueStick][start-server] Building server entry.
webpack building...

Starting to optimize CSS...
webpack built 030804976462ac97d2ed in 132ms
[GlueStick][start-server] File /Users/twilliams/dev/gs-v1/build/assets/webpack-chunks.json changed [change]
[GlueStick][start-server] Renderer listening on port 8880.

Starting to optimize CSS...
[GlueStick][start-server] Building server entry.
[GlueStick][start-server] Renderer listening on port 8880.

Starting to optimize CSS...
[GlueStick][start-server] Building server entry.

Starting to optimize CSS...
[GlueStick][start-server] Building server entry.

Starting to optimize CSS...
[GlueStick][start-server] Building server entry.

Starting to optimize CSS...
[GlueStick][start-server] Building server entry.

Starting to optimize CSS...
[GlueStick][start-server] Building server entry.
[GlueStick][start-server] Renderer listening on port 8880.
```

## Expected Result
I wouldn't expect things to rebundle so many times on a fresh startup in a new app when I'm haven't changed any files yet.

**Comment by zamotany** [comment]
This is a webpack issue, precisely watch feature of webpack. They use some screwed up solution, that detects changes to files by mistakes. That's why in `src/commands/start-server/runWithWebpack.js` we ditched `compiler.watch` in favour of `chokidar`.
So we have 2 options: add `aggregateTimeout` to sth like 1500-2000ms and we get usually 2 compilations, but then HMR will be way slower, because before building it will wait this specified amount of time. Second option is to rewrite `webpack-dev-middleware` but it would take some time.

**Comment by ferrannp** [comment]
This is resolved with `gluestick 1.1.0`.
